app/src/main/res/values/strings.xml

```diff
@@ -64,6 +64,7 @@
     <string name="audio_route_wired_headset">Wired Headset</string>
     <string name="audio_route_wired_or_earpiece">Wired or Earpiece</string>
     <string name="choose_audio_route">Choose audio route</string>
+    <string name="calling_blocked_numbe">The calling number is blocked</string>
```

Comment:
fix the typo "calling_blocked_numbe" to "calling_blocked_number" and change it to "The number you are calling is blocked"

Reply:
Thank you, I have fixed them